### PR TITLE
Feature/overview widget

### PR DESCRIPTION
**Pull request by tomek-ifm**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,3 +20,20 @@ metadata:
...
+```js
+// packages/app/src/components/catalog/EntityPage.tsx
+
+import { AWSLambdaOverviewWidget } from '@roadiehq/backstage-plugin-aws-lambda';
+
+const OverviewContent = ({ entity }: { entity: Entity }) => (
+  <Grid container spacing={3}>
+    ...
+    <Grid item>
+      <AWSLambdaOverviewWidget entity={entity} />
```
Comment (iain-b) [inline README.md:35]: Does the router export an isPluginApplicabletoEntity function ? worth documenting that I think. In this case where all components won't be lambdas users might want to hide that Grid element if the plugin isn't applicable (we explain the interface and let them use it as they see fit)

Comment (mcalus3) [reply README.md]: typescript highlights that you can import this function. But API documentation would be also nice. If we decide on it, I will also write it for travis-ci and firebase-functions plugins.

Comment (tomek-ifm) [reply README.md]: This is true, just commited this change @iain-b

File: package.json [hunk 2]
```diff
@@ -1,17 +1,18 @@
 {
   "name": "@roadiehq/backstage-plugin-aws-lambda",
-  "version": "0.1.1-alpha.16",
+  "version": "0.1.1",
```
Comment (iain-b) [inline package.json:3]: 0.2.0 ? Or did we bump already ?

**Review by iain-b — COMMENTED** [review]
Could we get some screenshots please? I know we've review later but it's good for reference

**Review by iain-b — APPROVED** [review]

File: src/components/AWSLambdaOverview/AWSLambdaOverview.tsx [hunk 3]
```diff
@@ -0,0 +1,235 @@
...
+  if (lambdaData.loading) {
+    console.log('loading');
+    return (
+      <Card>
+        <CardHeader title={<Typography variant="h5">AWS Lambda</Typography>} />
+        <LinearProgress />
+      </Card>
+    );
+  }
+  console.log('not loading');
```
Comment (mcalus3) [inline src/components/AWSLambdaOverview/AWSLambdaOverview.tsx:231]: console.log is here

**Pull request merged**